src/client/store/ethereum/actions.js

```diff
@@ -33,21 +33,12 @@ export function getTransactionId(txMethod, params) {
 
 export function initializeProvider() {
   return async (dispatch) => {
-    const provider = await detectMetaMask();
+    let provider = await detectMetaMask();
```

Comment:
`provider` does not get mutated, or why does this have to be `let`?

Reply:
doesn't. leftover from debugging ^